kitsune/sumo/static/sumo/js/instant_search.js

```diff
@@ -93,6 +101,8 @@
       });
     }
 
+    _gaq.push(['_trackEvent', 'Instant Search', 'Search', $this.data('href')]);
```

Comment:
There is some information about this here: https://support.google.com/analytics/answer/1012264?hl=en#Post

In particular, the event should be something like `ga('send', 'pageview', '/search?q=keyword');`